src/components/auth/torus/AuthTorus.js

```diff
@@ -182,8 +183,25 @@ const AuthTorus = ({ screenProps, navigation, styles, store }) => {
         throw new Error('Missing privateKey from torus response')
       }
 
+      let masterSeed = torusUser.privateKey
+
       //set masterseed so wallet can use it in 'ready' where we check if user exists
-      await AsyncStorage.setItem(GD_USER_MASTERSEED, torusUser.privateKey)
+      // Initialize the sdk here
+      if (config.ceramicEnabled) {
```

Comment:
this is a code repetition, we could re-use useCeramicSDK hook here (`getMasterSeed()` from its adjusted version)